tests/javaRosa.js

```diff
@@ -366,8 +366,8 @@ define([
 
         it("itext widget should not overwrite label with question id", function () {
             util.loadXML("");
-            var q1 = util.addQuestion("Text");
-            util.addQuestion("Text");
+            var q1 = util.addQuestion("Text", "question1");
+            util.addQuestion("Text", "question2");
```

Comment:
> Is it now required to always pass a nodeID to `addQuestion()`?

Not exactly, but it's harder to predict when the id will be generated. It does make sense that 99% of the time tests will want the id to be generated by the time `addQuestion` returns. Updated the `addQuestion` in `tests/utils` to always generate an id.